R/methods.R

```diff
@@ -11,34 +10,36 @@ print.lint <- function(x, ...) {
   cat(
     sep = "",
     crayon::bold(x$filename, ":",
-    as.character(x$line_number), ":",
-    as.character(x$column_number), ": ", sep = ""),
+      as.character(x$line_number), ":",
+      as.character(x$column_number), ": ",
+      sep = ""
```

Comment:
is sep= used twice here?

Reply:
Btw, I am laughing my ass off at this comment  🤣

```r
# swap tabs for spaces for #528 (sorry Richard Hendricks)
```

Reply:
oh, I see it more clearly now that the blear's gone, thanks!